### DISCUSSION TITLE
Dropdown loading and footer template add

### DISCUSSION THREAD

**Discussion by serdargoleli** [primefaces/community#127] [PrimeReact] [answered]
_Moderator edit for English translation:_

We feel the lack of loading and footer template props in the PrimeReact > Dropdown component, will there be development regarding this?

Thank you

---

primereact > dropdown bileşeninde loading ve footer template propslarının eksikliğini hissediyoruz, bununla ilgili bir geliştirme yapılacak mıdır? 

Teşekkürler

**Comment by habubey** [comment, marked as answer]
I've opened a [ticket](https://github.com/primefaces/primereact/issues/5005) for this new feature request. You can follow updates on the new feature's development here.
Thanks for the suggestion.

**Comment by mertsincan** [comment]
Hi,

Sorry for the delayed response! We can also expect upvotes from other users. We will be happy to implement it when we get enough upvotes. Thanks a lot for your understanding!
